index.bs

```diff
@@ -3657,7 +3664,91 @@ This is because the first 37 bytes of the signed data in a FIDO U2F authenticati
 <code>[=authDataExtensions|extensions=]</code> are never present. FIDO U2F authentication signatures can therefore be verified by
 the same procedure as other [=assertion signatures=] generated by the [=authenticatorGetAssertion=] operation.
 
+### Credential Backup State ### {#sctn-credential-backup}
 
+Credential backup eligibility and current backup state is conveyed by [=Authenticator data|authenticator data=] bits `3` and `4` as 
+defined in <a href="#table-authData">Table <span class="table-ref-following"/></a>.
+
+The value of bit 3 is set during [=authenticatorMakeCredential=] operation and MUST NOT change. 
```

Comment:
> It would be good to elaborate on or refer to a section where it is explained how the authenticator determines the value of that bit in authenticatorMakeCredential.
> For example, refer to DPK extension, or similar.

@rlin1 the authenticator is ultimately responsible for the lifecycle management and state of the credential. I'm not sure how we would put the internal business logic of an authenticator into the spec. Do you have any suggested text?

Reply:
I was not referring to business logic.  But in PR#1663 it is kind of obvious that the RP could add the DPK extension to a getAssertion call in order to ask for a non-synchronizable key.
In this Issue 1692 - when using the flags approach it is not clear (to me) how the RP could ask for a singleDeviceKey.
So we should add an explanation to this PR1695 how the RP could do that - e.g. using the  credProps or using some extension in the request - or clarify that the RP cannot (if that is the intention)